app/controllers/general.php

```diff
@@ -300,6 +301,35 @@
 
                 Authorization::setRole('role:' . Auth::USER_ROLE_APP);
                 Authorization::setDefaultStatus(false);  // Cancel security segmentation for API keys.
+
+                if (time() > ($key->getAttribute('accessedAt', 0) + (24 * 60 * 60))) {
```

Comment:
Let's use a constant for the 24 hours range. We might also want to wait for the dates PR.

Reply:
done 👍